robot/Cumulus/perf/BDITest.robot

```diff
@@ -0,0 +1,8 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+
+*** Test Cases ***
+
+Do a Batch Load via API
+    Api Batchdata Load
```

Comment:
It looks to me like you can use the existing `Run Task Class` keyword rather than adding a new keyword:

    Run Task Class | cumulusci.tasks.apex.anon.AnonymousApexTask | apex=BDI_DataImport_API.importData();
    Run Task Class | cumulusci.tasks.apex.batch.BatchApexWait | class_name= BDI_DataImport_BATCH

Reply:
That's true, but is it necessarily better? Do we want Apex code in Robot code? Seems like its punching through layers to me, though I don't feel strongly about it.

Also: I suspect its going to get more complicated in the future. Dry Runs? Field Mappings? If so, it might be easier to maintain in Python.

Reply:
I can see a Python task making more sense if it does get more complicated.

Taking a look at the big picture, it seems dumb to me that we have to use Robot Framework for this at all, just to track the times. We're basically just running a flow. Maybe MetaCI needs a way to record times for the tasks in the flows it runs or something like that.